### DISCUSSION TITLE
Moving "schemas"-folder into asyncua?

### DISCUSSION THREAD

**Discussion by swamper123** [FreeOpcUa/opcua-asyncio#477] [Ideas]
In case python-asyncio is installed via pip for a project, you wouldn't be able to update the UA Nodeset via update_ua_nodeset.py (because this wouldn't exist).
So in case the foundation updates the Nodeset and we don't push a new release with the updated nodeset, a user may not be able to import required Nodesets. 

Should it be moved into "asyncua" or am I missing something?

**Comment by oroulet** [comment]
We do not want for sure to distribute the nodesets with asyncua. So no it should not go into asyncua  folder.
I think for now it is fine to realease a new version. That is what everyone does (plc suppliers, machine suppliers, etc...)
